### DISCUSSION TITLE
Github requires me to apply through a school issued email but my school/clg doen't provide any email, even the listed email is wrong

### DISCUSSION THREAD

**Discussion by gautam8404** [community/community#63926] [GitHub Education]
### Hi everyone,

Github requires me to apply through a school issued email for my college but our college doesn't provide any email to students
![image](https://github.com/community/community/assets/93448885/e84ce376-bef8-4f6d-a84f-1ff63401ac27)

even the email mentioned is 'gniotmba.net' which is for MBA a non related course

i've tried opening tickets #2261306 and #2269429 to edit school info no response there too

**Comment by safnimj** [comment]
If your Institution was not in List You can add it by contacting a Github Campus Expert

**Reply by gautam8404** [reply]
It's in list but that's te problem, the info is wrong as our college do not provide us any email

**Comment by gautam8404** [comment]
Hey @samus-aran 

I've tried almost about everything nothing seems to work do you have any idea how can I reverify myself? My dev pack has expired today I've already opened 2 tickets to edit my uni info about 1 month ago no reply there too

![Screenshot_2023-08-21-21-48-12-46_e307a3f9df9f380ebaf106e1dc980bb6.jpg](https://github.com/community/community/assets/93448885/14beabe4-d03b-43e9-bdc3-3abb614f327a)
